openstack/utils/templates/snippets/_proxysql.cfg.tpl

```diff
@@ -49,12 +59,75 @@ mysql_variables =
 mysql_servers =
 (
 {{- range $index, $dbKey:= $.dbKeys }}
-  {{- $db := get $dbs $dbKey }}
+{{- /*
+Example for Galera cluster:
+    {
+        address = "test-db-pxc-0.domain.svc.cluster.local"
+        hostgroup = 11
+        max_connections = 20
+        weight = 101
+    }
+*/}}
+    {{- $db := get $dbs $dbKey }}
+    {{- if hasKey $db "pxc" }}
+    {{- range $i := until (int $db.pxc.size) }}
+    {
+        address = "{{ $db.name }}-db-pxc-{{ $i }}.{{ include "svc_fqdn" $.global }}"
+        hostgroup = {{ add 1 (mul 10 (add1 $index)) }}
```

Comment:
The implication here is that `$db.pxc.size < 10`, correct?
I'd at least test that and raise an error, if not.

Reply:
In general, a cluster shouldn't be bigger than 7 members.
I will add a check to pxc-db chart and here.